assets/test/SignPanel.test.ts

```diff
@@ -599,3 +599,49 @@ test('does not save changes to backend until Apply is pressed', () => {
     },
   ]);
 });
+
+test("shows Apply button but not expiration form for signs with no 'Auto' mode", () => {
```

Comment:
Since this actually tests the usage of the Apply button rather than checking whether it's there, maybe `"allows setting custom text for signs with no Auto mode"`?